rancher/metadata-service/index.md

```diff
@@ -111,14 +163,18 @@ $ curl --header 'Accept: application/json' 'http://rancher-metadata/2015-07-25/s
 | `primary_ip` | IP of container
 | `service_name` | Name of service (if applicable)
 | `stack_name` | Name of stack that the service is in (if applicable)
-
+| `start_count` | The number of times the container was started.
+| `uuid` | Unique container identifier that Rancher assigns to containers
 
 ### Service
 
  Fields | Description
 ----|----
 `containers` | List of container names in the service
+`create_index` | The order number of which the service was launched in the stack, i.e. 2 means it was the second service launched in the stack.
```

Comment:
not exactly right. create_index reflects number of containers that were ever deployed in the service, or in other words, the create_index on the last service created. For example, service with scale=3 once activated, will have 3 containers each having:

c1 create_index=1
c2 create_index=2
c3 create_index=3

service will end up having create_index=3.

Then lets say c3 gets removed, and c4 gets created. WE never reuse create_index, so C4 will get create_index=4, and service will reflect this number as well.